ivy-test-cluster-poject/pom.xml

```diff
@@ -7,20 +7,34 @@
   <version>1.0.0-SNAPSHOT</version>
   <packaging>iar</packaging>
   <properties>
-    <engine.page.url>https://jenkins.ivyteam.io/job/core_product/job/master/lastSuccessfulBuild/</engine.page.url>
+    <engine.page.url>https://product.ivyteam.io/</engine.page.url>
+    <project.build.plugin.version>11.1.0-SNAPSHOT</project.build.plugin.version>
   </properties>
   <build>
     <plugins>
       <plugin>
         <groupId>com.axonivy.ivy.ci</groupId>
         <artifactId>project-build-plugin</artifactId>
-        <version>10.0.0</version>
+        <version>${project.build.plugin.version}</version>
         <extensions>true</extensions>
         <configuration>
-          <ivyVersion>[10.0,]</ivyVersion>
+          <ivyVersion>[11.1,]</ivyVersion>
           <engineListPageUrl>${engine.page.url}</engineListPageUrl>
         </configuration>
       </plugin>
     </plugins>
   </build>
+
+  <pluginRepositories>
```

Comment:
Because project build plugin has SNAPSHOT version?

Reply:
Yes if we work with snapshot versions, this is needed to find them...